packages/framework/src/Model/Complaint/ComplaintFactory.php

```diff
@@ -17,13 +17,15 @@ public function __construct(protected readonly EntityNameResolver $entityNameRes
 
     /**
      * @param \Shopsys\FrameworkBundle\Model\Complaint\ComplaintData $complaintData
+     * @param array $complaintItems
      * @return \Shopsys\FrameworkBundle\Model\Complaint\Complaint
      */
     public function create(
         ComplaintData $complaintData,
+        array $complaintItems,
```

Comment:
I originally set it up that way, but I changed it because I also need to manage files. Since the files aren't directly associated with the OrderItem entity, it would require passing them around separately.

Reply:
ok :+1: